atomic_reactor/plugins/post_koji_upload.py

```diff
@@ -444,6 +444,25 @@ def add_log_type(output, arch):
 
         return output_files
 
+    def update_buildroot_koji(self, buildroot, output):
+        """
+        put the final koji information in the buildroot under extra.osbs
+        """
+        docker = output[1]['extra']['docker']
+
+        name = ''
+        for tag in docker['tags']:
+            for repo in docker['repositories']:
+                if tag in repo:
+                    iname = ImageName.parse(repo)
+                    name = iname.to_str(registry=False)
+                    break
+
+        buildroot['extra']['osbs']['koji'] = {
+            'build_name': name,
+            'builder_image_id': docker.get('digests', [])
```

Comment:
Should the default value be `{}` instead?

Reply:
Fixed.